Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM node:8.11.2-stretch
+FROM node:10.14.0-stretch
 
 LABEL maintainer="Lucas Clay <lclay@shipchain.io>"
 
 ENV LANG C.UTF-8
 ENV PYTHONUNBUFFERED 1
 
-RUN apt-get update -y && apt-get install -y libgmp3-dev libstdc++6 jq
```

Comment:
We're using jq in our entrypoint script - this deletion will break our deployed environments. Not sure about the other two libs.

Reply:
Good catch!  I only looked at the `*entrypoint.sh` scripts.  Not the `download-certs.sh` one.